src/main/target/EXF722DUAL/target.h

```diff
@@ -35,7 +35,7 @@
 #define USE_EXTI
 #define USE_GYRO_EXTI
 #define GYRO_1_EXTI_PIN         PA8
-#define GYRO_2_EXTI_PIN         PC13
```

Comment:
Were there any boards using this EXTI pin ever produced / sold?

Reply:
Can you please answer this?

Reply:
@mikeller Thanks for your reminder. Please forgive me for not telling things clearly. 
Yes, the IMU2's EXTI pin of the EXF722 Flight Controller board is always PB02, but it's just that when I submitted the target before, I coded the wrong pin number.
This is a bug that has been fixed here and I hope to get the code merge as soon as possible.